src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -46,60 +48,30 @@ public class RazorViewEngine : IRazorViewEngine
             IRazorPageFactoryProvider pageFactory,
             IRazorPageActivator pageActivator,
             HtmlEncoder htmlEncoder,
-            IOptions<RazorViewEngineOptions> optionsAccessor,
+            IOptions<RazorViewEngineOptions> options,
             ILoggerFactory loggerFactory)
         {
             _pageFactory = pageFactory;
             _pageActivator = pageActivator;
-            _viewLocationExpanders = optionsAccessor.Value.ViewLocationExpanders;
             _htmlEncoder = htmlEncoder;
             _logger = loggerFactory.CreateLogger<RazorViewEngine>();
             ViewLookupCache = new MemoryCache(new MemoryCacheOptions
             {
                 CompactOnMemoryPressure = false
             });
-        }
 
-        /// <summary>
-        /// Gets the locations where this instance of <see cref="RazorViewEngine"/> will search for views.
-        /// </summary>
-        /// <remarks>
-        /// The locations of the views returned from controllers that do not belong to an area.
-        /// Locations are composite format strings (see http://msdn.microsoft.com/en-us/library/txafckwd.aspx),
-        /// which contains following indexes:
-        /// {0} - Action Name
-        /// {1} - Controller Name
-        /// The values for these locations are case-sensitive on case-sensitive file systems.
-        /// For example, the view for the <c>Test</c> action of <c>HomeController</c> should be located at
-        /// <c>/Views/Home/Test.cshtml</c>. Locations such as <c>/views/home/test.cshtml</c> would not be discovered
-        /// </remarks>
-        public virtual IEnumerable<string> ViewLocationFormats { get; } = new[]
```

Comment:
> What other possible use is there?

I said "subclass" in my most recent comment. But I understand your refutation of my more general "available to anything that wanted to _read_ them" comment above.